### ISSUE TITLE
Autocomplete based on current working directory

### ISSUE THREAD

**Issue by willfong** [warpdotdev/Warp#1290]
### Describe the solution you'd like?

```text
The current algorithm for auto completion doesn't seem to take into consideration the current working directory. For example, I have two projects:

~/Projects/first
~/Projects/second

Let's say in project "first", I normally use "yarn start" while in project "second", I would use "yarn dev". However, auto complete suggestions will suggest the other project's "yarn" completion.  It should recognize that when I'm in "first", give me "yarn start", instead of "yarn dev".

Thanks!
```

**Comment by dpgraham4401** [comment]
Yes! 100%. I'm loving the look, and dropdown ability, and other functionality of warp, but it appears that the completion suggestions just completely disregards files/directories in my current working directory. 

Example
``` 
mv foobar.html foo.html
```
if I decide to rename this file again, and I type `mv f` instaed of autocomplete suggesting `mv foo.html` (the name of the only file or directory starting with an "f" in my cwd) it suggests `mv foobar.html foo.html` again. or it will give me some old command that is irrelevant.

Also, tab/completion seems like it should only complete as much as makes sense. 
Example:
If we're in a hypothetical directory with the following (horribly named) files
- manifest.yaml
- manifest_svc.yaml
- manifest_ingress.yaml
if I try `vim man` I only want warp to autocomplete `vim manifest`, instead of auto completing to a previous command like `vim manifest_ingress.yaml`. Cause now, I gotta spam the delete button and manually finish `vim manifest_svc.yaml`. 

This is kinda makes it impossible to quickly navigate via the terminal.

**Comment by elviskahoro** [comment]
Thanks for opening this! Expect improvements over the next 2 quarters!

**Comment by EasyAsABC123** [comment]
@elviskahoro 
To piggyback on this
```bash
python3 filename.py
```
filename.py isn't being autocompleted for me...which is really painful.

**Comment by steffenrog** [comment]
Any updates on this?

**Comment by iamef** [comment]
Still having this issue. 

Turning on and off does not help for me.
 Preferences -> Settings -> Features -> Open completions menu as you type

**Comment by zen-dog** [comment]
Same issue here: with and without `Preferences -> Settings -> Features -> Open completions menu as you type`, auto complete seems to ignore file names in basic bash commands. I like the smart aspects of the auto-complete but having to manually type in file names isn't very smart.

**Comment by chenyukang** [comment]
Still having this issue, emmm.
